### ISSUE TITLE
Not able to automate the app due to missing "accessibilityLabel" or "testID"

### ISSUE THREAD

**Issue by sasasa42** [GeekyAnts/NativeBase#1352]
## react-native, react and native-base version
"native-base": "^2.3.2",
"react-native": "^0.49.1"
## Expected behaviour
Expecting on Android Inspector to show the "content-desc" in the node view
## Actual behaviour
The "content-desc" is blank.
## Steps to reproduce (code snippet or screenshot)

1. Open the react native app on an android device
2. Use Appium or "uiautomatorviewer" to inspect the elements
3. The react-native elements show up at least the "content-desc" through the "accessibilityLabel" attribute, but the native-base elements does not show that either.

## Is the bug present in both ios and android or in any one of them?
The issue is seen for both "ios" and "android"

## Any other additional info which would help us debug the issue quicker.
It would be good if we can get "accessibilityid" for every elements in the app through either "content-desc" or "resource-id"

**Comment by akhil-geekyants** [comment]
See issue #1230. Similar requirements
